src/layers/dist.rs

```diff
@@ -49,7 +49,7 @@ impl Layer for DistLayer {
         tgz::fetch_strip_filter_extract_verify(
             self.artifact.mirror_tarball_url(),
             "go",
-            ["bin", "src", "pkg", "LICENSE"].into_iter(),
+            ["bin", "src", "pkg", "go.env", "LICENSE"].into_iter(),
```

Comment:
If we change the interface of `fetch_strip_filter_extract_verify` to take an `IntoIterator` instead of an `Iterator` then you can drop that `into_iter()` call:

```
            ["bin", "src", "pkg", "go.env", "LICENSE"],
```